common/translations/lang/en.json

```diff
@@ -909,6 +909,10 @@
     "TRUST_DOWNLOAD": "Download Trust",
     "KEYSTORE_HELP": "Not working? [Here's some troubleshooting tips to try](https://support.mycrypto.com/how-to/accessing-wallet/how-to-access-your-wallet-with-keystore-file)",
     "MNEMONIC_HELP": "Not working? [Here's some troubleshooting tips to try](https://support.mycrypto.com/how-to/accessing-wallet/how-to-access-your-wallet-with-mnemonic-phrase)",
+    "LEDGER_NANO_S_HELP": "Not working? [Here's some troubleshooting tips to try](https://support.mycrypto.com/troubleshooting/accessing-wallet/ledger-hardware-wallet-unable-to-connect-on-mycrypto)",
```

Comment:
@blurpesec @FrederikBolding 
FYI: The link urls are indifferent to the language.
Instead of having to copy paste them throughout our translation files, use the `$variable` syntax.